k-distribution/tests/regression-new/map-symbolic-tests-haskell/remove-2-spec.k

```diff
@@ -6,6 +6,6 @@ requires "map-tests.k"
 module REMOVE-2-SPEC
     imports MAP-TESTS
 
-    claim <k> remove ( (MAP:Map X:MyId |-> 1) [Y:MyId <- undef] [ Z:MyId ] ) => removeResult ( MAP [ Z ] ) </k> requires Y ==K X andBool Z  ==K Y andBool Z  ==K X
+    claim <k> remove ( (MAP:Map X:MyId |-> 1) [Y:MyId <- undef] [ Z:MyId ] ) => . </k> requires Y ==K X andBool Z  ==K Y andBool Z  ==K X
```

Comment:
The output is still `#Top` here, and I'm not sure if this is right.

Reply:
That is correct. All the keys are the same. Looking up the key after deleting it is `\bottom`.